plugins/labormanager.cpp

```diff
@@ -673,8 +673,15 @@ static df::building* get_building_from_job(df::job* j)
     return 0;
 }
 
-static df::unit_labor construction_build_labor (df::item* i)
+static df::unit_labor construction_build_labor (df::building_actual* b)
 {
+    if (b->getType() == df::building_type::RoadPaved)
+        return df::unit_labor::BUILD_ROAD;
+    auto a = (df::building_actual *) b;
```

Comment:
This patch changed it to a df::item. It'll actually be a building_actual at that point, but C++ is too stupid to know this.